### PR TITLE
[JBWS-4167] add archives so project compiles with jdk-10+

### PR DESCRIPTION
**Pull request by rsearls**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -61,7 +66,29 @@
...
+    <dependency>
+      <groupId>org.jboss.spec.javax.xml.bind</groupId>
+      <artifactId>jboss-jaxb-api_2.3_spec</artifactId>
+      <version>${version.org.jboss.spec.javax.xml.bind-api_2.3_spec}</version>
+    </dependency>
+    <dependency>
+      <groupId>javax.xml.ws</groupId>
+      <artifactId>jaxws-api</artifactId>
+      <version>${version.jaxws-api}</version>
+    </dependency>
```
Comment (asoldano) [inline pom.xml:90]: I guess this is not needed, as there's already a dependency to the JAX-WS API 2.3 above.

**Review by asoldano — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (rsearls) [reply pom.xml:90]: This archive is required.  There are references to package javax.jws  that this archive provides.

Comment (asoldano) [reply pom.xml:90]: the javax.jws package comes from JSR181 API, javax.jws:jsr181-api, which is a transitive dependency of javax.xml.ws:jaxws-api and is what is most likely providing the stuff you need here; I'd say check which GAV we currently use in JBossWS for JSR181 API and include that instead. javax.xml.ws:jaxws-api shouldn't be included given we have our fork that you're correctly already pulling.

File: pom.xml [hunk 2]
```diff
@@ -31,6 +31,12 @@
     <cxf.version>2.7.13</cxf.version>
     <servlet-api.version>1.0.2.Final</servlet-api.version>
     <io.undertow.version>1.0.10.Final</io.undertow.version>
+    <jaxws.api.version>1.0.0.Final</jaxws.api.version>
+    <version.javax.activation>1.2.0</version.javax.activation>
+    <version.org.jboss.spec.javax.annotation.jboss-annotations-api_1.3_spec>1.0.1.Final</version.org.jboss.spec.javax.annotation.jboss-annotations-api_1.3_spec>
+    <version.org.jboss.spec.javax.xml.bind-api_2.3_spec>1.0.1.Final</version.org.jboss.spec.javax.xml.bind-api_2.3_spec>
+     <version.jaxws-api>2.2.6</version.jaxws-api>
```
Comment (jimma) [inline pom.xml]: @rsearls   can jaxws-api version properity be removed too ?

**Review by jimma — COMMENTED** [review]
@rsearls When I tested with jdk11 , I saw couple of test failures. Looks we have to fix more things for jdk11:
- Remove endorse compile with a maven profile ? 
- ~~Move to CXF 3.2.6~~
- ~~Upgrade undertow version to 2.0.13.Final~~
- ~~Fix test failures~~
I fixed the last 3 items with PR https://github.com/jbossws/jaxws-undertow-httpspi/pull/4 for jdk8 and this should work for jdk11 too.  Please review.

**Comment by rsearls** [comment]
done

**Pull request merged**